gen-certs

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright 2025 Contributors to the Veraison project.
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+ROOT_CERT_NAME=rootCA
+DEFAULT_DOMAIN=veraison-net
+DEFAULT_SUFFIX=-ratsd
+
+SERVICE=(ratsd)
+THIS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+function create_root_cert() {
+	_check_openssl
+	pushd "$THIS_DIR" > /dev/null || exit 1
+
+	openssl ecparam -name prime256v1 -genkey -noout -out ${ROOT_CERT_NAME}.key
+	openssl req -x509 -new -nodes -key ${ROOT_CERT_NAME}.key -sha256 -days 3650 \
+		-subj "/O=Veraison" -out ${ROOT_CERT_NAME}.crt
+
+	echo "Created ${THIS_DIR}/${ROOT_CERT_NAME}.crt"
+
+	popd > /dev/null || exit 1
+}
+
+function create_service_cert() {
+	_check_openssl
+	_check_root_cert
+	pushd "$THIS_DIR" > /dev/null || exit 1
+
+	local suffix=$1
+	local domain=$2
+	local force=$3
+	local san
+
+	san=$(printf "subjectAltName = DNS:%s%s.%s,DNS:%s%s,DNS:localhost" \
```

Comment:
The SAN should ideally have the hostname and IP address of the server running the server instead of localhost, but it's not possible know the IP of the server during build time.

Reply:
Thanks @cowbon ! The PR looks good to me.

Please address comments from @thomas-fossati and @iolivergithub  before merging your PR.

Reply:
> The SAN should ideally have the hostname and IP address of the server running the server instead of localhost, but it's not possible know the IP of the server during build time.

In general, it's better to use DNS names rather than IP addresses in the SAN.  You let then the resolver do the job it's paid for :-)

Using `localhost` among the lot is OK, especially for a cert that is going to be used in a dev environment like this one.